.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish to PyPI and TestPyPI
+
+on:
+  # Trigger the workflow on push, but only for the master branch
+  push:
+    branches:
+      - master
```

Comment:
Do you have a sense of whether this will also run for tagged releases? It's OK if not, I wouldn't be surprised if this wasn't well-documented yet. If that's the case I'm comfortable merging and trying it and then adjusting the Action if we need to.

Reply:
My guess is yes but I wasn't able to find conclusive documentation. I'm going to go ahead and try!